pkg/deployments/index.ts

```diff
@@ -92,3 +103,11 @@ function getBalancerContractArtifactPath(task: string, contract: string): string
 function getBalancerDeploymentPath(task: string, network: string): string {
   return `@balancer-labs/v2-deployments/dist/tasks/${task}/output/${network}.json`;
 }
+
+/**
+ * @dev Returns the path for the list of Balancer contract addresses on a network
+ * @param network Name of the network looking the deployment path for (e.g. mainnet, polygon, goerli, etc)
+ */
+function getBalancerContractAddresses(network: string): string {
+  return `@balancer-labs/v2-deployments/dist/addresses/${network}.json`;
+}
```

Comment:
What's the purpose of this function?

Reply:
It returns the path of the file which contains the addresses on a particular network. It means so that if the path changes in future we only need to change it in a single place.